server/api/src/services/map.py

```diff
@@ -105,6 +134,7 @@ async def heatmap(startDate,
     return pins.to_numpy()
 
 
+# seems to be unused. remove?
 async def pins(startDate,
```

Comment:
I just added this for Adam, who may be using it in his updates to the map to use pins instead of clusters. So let's leave it in until we see what happens with the clusters.

Reply:
got it. thanks.